index.html

```diff
@@ -161,123 +114,130 @@ <h2>
+          <li>Return <strong>undefined</strong>.</li>
+        </ol>
+      </section>
+      <section>
+        <h2><dfn>clearMarks()</dfn> method</h2>
+        <p>Removes the stored timestamp with the associated name. It MUST run these steps:</p>
+        <ol>
+          <li>If <var>markName</var> is omitted, remove all <a>PerformanceMark</a> objects from the <a data-cite="!PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer">performance entry buffer</a>.</li>
+          <li>Otherwise, remove all <a>PerformanceMark</a> objects listed in the <a data-cite="!PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer">performance entry buffer</a> whose <var>name</var> matches<var>markName</var>.</li>
+          <li>Return <strong>undefined</strong>.</li>
+        </ol>
+      </section>
+      <section>
+        <h2><dfn>measure()</dfn> method</h2>
+        <p>Stores the <code>DOMHighResTimeStamp</code> duration between two marks along with the associated name (a "measure"). It MUST run these steps:</p>
+        <ol>
+          <li>Run the <a>ensure name is valid</a> algorithm with <var>name</var> set to <var>measureName</var>.</li>
+          <li>Let <var>end time</var> be <code>0</code>.</li>
+          <li>If <var>endMark</var> is omitted, let <var>end time</var> be the value that would be returned by the <code>Performance</code> object's <a data-cite="!HR-TIME-2#dom-performance-now"><code>now()</code></a> method. Otherwise:
+            <ol>
+              <li>If <var>endMark</var> has the same name as a <a data-cite="!WEBIDL#dfn-read-only">read only attribute</a> in the <code><a data-cite="!NAVIGATION-TIMING#performancetiming">PerformanceTiming</a></code> interface, let <var>end time</var> be the value returned by running the <a>convert a name to a timestamp</a> algorithm with <var>name</var> set to the value of <var>endMark</var>.</li>
+              <li>Otherwise let <var>end time</var> be the value of the <code>startTime</code> attribute from the most recent occurrence of a <a>PerformanceMark</a> object in the <a data-cite="!PERFORMANCE-TIMELINE-2#dfn-performance-entry-buffer">performance entry buffer</a> whose <code>name</code> matches the value of <var>endMark</var>. If no matching entry is found, <a data-cite="!WEBIDL#dfn-throw">throw</a> a <a data-cite="!WEBIDL#syntaxerror"><code>SyntaxError</code></a>.</li>
+            </ol>
+          </li>
+          <li>If <var>startMark</var> is omitted, let <var>start time</var> be <code>0</code>. Otherwise:
+            <ol>
+              <li>If <var>startMark</var> has the same name as a <a data-cite="!WEBIDL#dfn-read-only">read only attribute</a> in the <code><a data-cite="!NAVIGATION-TIMING#performancetiming">PerformanceTiming</a></code> interface, let <var>start time</var> be the value return by running the <a>convert a name to a timestamp</a> algorithm with <var>name</var> set to <var>startMark</var>.</li>

```

Comment:
Same as above.

Reply:
I believe so. This matches current implementations and tests as mentioned in #21.